.circleci/config.yml

```diff
@@ -0,0 +1,145 @@
+version: 2
+
+workflows:
+  version: 2
+  kubeapps:
+    jobs:
+      - test_go
+      - test_dashboard
+      - build_kubeapps_cli
+      - build_chartrepo:
+          requires:
+            - test_go
+      - build_chartsvc:
+          requires:
+            - test_go
+      - build_chart_apprepository:
+          requires:
+            - test_go
+      - build_dashboard:
+          requires:
+            - test_dashboard
+      - build_tiller_proxy:
+          requires:
+            - test_go
+      - release:
+          requires:
+            - build_chartrepo
+            - build_chartsvc
+            - build_chart_apprepository
+            - build_dashboard
+            - build_tiller_proxy
+
+## Definitions
+install_gcloud_sdk: &install_gcloud_sdk
```

Comment:
Based on our conversation this morning, I think that we should remove any reference to the CLI here and its dependencies.

WDYT?

Reply:
:+1:

Reply:
I removed that